css-responsive/css-responsive_playground/css/styles.css

```diff
@@ -0,0 +1,121 @@
+/* ==[ PLAYGROUND Challenge! ]== */
+
+/* This challenge includes several smaller challenges and allows you to
+play around with the concepts you've learned. Read the instructions 
+carefully. */
+
+:root {
+  /* Color Variables */
+
+  --color-nemo: #ff4a11;
+  --color-granite: #252629;
+  --color-water-10: #f3f5f9;
+  --color-foam: #ffffff;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: system-ui;
+
+  background-color: var(--color-foam);
+}
+
+h1 {
+  font-size: 3rem;
+  width: 80%;
+  margin: 20px auto;
+  color: var(--color-nemo);
+  text-shadow: 4px 4px var(--color-granite);
+}
+
+.playground {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: auto;
+  padding: 20px;
+  gap: 30px;
+
+  width: 80%;
+  min-height: 100vh;
+}
+
+.section {
+  width: 100%;
+  background-color: var(--color-water-10);
+  padding: 20px;
+}
+
+.box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+}
+
+.box--one {
+  background-color: lightblue;
+
+  width: 100%;
+  height: 100px;
+}
+
+.box--two {
+  background-color: lightgreen;
+  width: 100px;
+  height: 100px;
+}
+
+.box--three {
+  background-color: lightpink;
+
+  width: 50px;
+  height: 250px;
+}
+
+.box--four {
+  background-color: lightgray;
+
+  width: 100px;
+  height: 100px;
+}
+
+.box--five {
+  width: 100%;
+}
+
+.box--five img {
+  width: 100%;
+}
+
+@media screen and (min-width: 599px) {
```

Comment:
This is a little one. It says *"wider than 599px"*. So *600px* would be correct here.